src/test/java/step2/Calculator.java

```diff
@@ -0,0 +1,68 @@
+package step2;
+
+import org.junit.jupiter.params.ParameterizedTest;
+import org.junit.jupiter.params.provider.ValueSource;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+public class Calculator {
+
+    int stringCalculatingProcessor(String rawInput) {
```

Comment:
접근 제어자를 올바르게 명시해 주시면 좋겠습니다. 
이 메소드는 `public` 이여야 할 것 같은데요. 

```suggestion
    public int stringCalculatingProcessor(String rawInput) {

Reply:
- [ ] 접근 제한자 명시하기